README.md

```diff
@@ -1,149 +1,17 @@
 [![Pylint](https://github.com/snu-quiqcl/swift/actions/workflows/pylint.yml/badge.svg)](https://github.com/snu-quiqcl/swift/actions/workflows/pylint.yml)
 
 # swift
-**S**NU **w**idget **i**ntegration **f**ramework for PyQ**t**
+Swift (**S**NU **w**idget **i**ntegration **f**ramework for PyQ**t**) is a framework for integration of PyQt widgets where they can communicate with each other. This project is mainly developed for trapped ion experiment controller GUI in SNU QuIQCL.
 
-A framework for integration of PyQt widgets where they can communicate with each other.
-This project is mainly developed for trapped ion experiment controller GUI in SNU QuIQCL.
+## How to download
+One of the below:
+- git clone or fetch
+- Download the latest release: [v1.0.1](https://github.com/snu-quiqcl/swift/releases/tag/v1.0.1)
 
-`swift` provides a dashboard-like main window, in which variety of `Frame`s reside.
-A `Frame` is in fact a special `QWidget` which obeys the interface of `swift`, and it will be wrapped by a `QDockWidget`.
+For convenience, you may download test databases:
+- [database.zip](https://github.com/snu-quiqcl/swift/files/10907031/database.zip)
```

Comment:
Please add more comments on what this is.
Actually, since the example is not a part of the package in fact, I think this should be in example section somewhere in the Wiki.
What do you think of this? Should it be here in README.md?

Reply:
I agree with you.

Then, how about moving to the discussion? Actually, it's already there.

Reply:
> Then, how about moving to the discussion? Actually, it's already there.

Great, then let's remove it from README.md.